provider/redis.go

```diff
@@ -184,6 +184,18 @@ func (store *redisOnlineStore) DeleteTable(feature, variant string) error {
 	return nil
 }
 
+func (store *redisOnlineStore) Check() (bool, error) {
```

Comment:
I think it should be out of scope but we should add more robust checks later on to make sure we have read and write access to the providers.